api/docs/CMake_rundoxygen.cmake

```diff
@@ -185,7 +185,12 @@ if (embeddable)
         "<object type=\"image/svg\\+xml\" data=\""
         "<object type=\"image/svg+xml\" data=\"/images/"
         string "${string}")
-
+    # Ensure that strings that look like Jekyll tags are not interpreted as such.
+    # We add the Jekyll raw-endraw tags around "{%" and "%}" separately.
+    string(REGEX REPLACE "{%" "jekyll_raw_tag{%jekyll_endraw_tag" string "${string}")
+    string(REGEX REPLACE "%}" "jekyll_raw_tag%}jekyll_endraw_tag" string "${string}")
+    string(REGEX REPLACE "jekyll_raw_tag" "{% raw %}" string "${string}")
```

Comment:
Thinking whether I should throw a random number in there `jekyll_raw_tag_1234` or something. So that it doesn't match with actual text if we ever add it to doc.

Reply:
OK, or a bunch of underscores or sthg.

Reply:
Done.